pkg/infra/httpclient/httpclientprovider/http_client_provider_test.go

```diff
@@ -87,3 +97,135 @@ func TestHTTPClientProvider(t *testing.T) {
 		require.Equal(t, HTTPLoggerMiddlewareName, o.Middlewares[8].(sdkhttpclient.MiddlewareName).MiddlewareName())
 	})
 }
+
+func TestNewSecureSocksProxy(t *testing.T) {
```

Comment:
Nit. If moving `newSecureSocksProxy` to a new file would move this to a separate test file.

Reply:
Done 👍